xfstests/local.config

```diff
@@ -7,6 +7,6 @@ export FUSE_SUBTYP=.passthrough
 export MOUNT_OPTIONS=""
 export TEST_FS_MOUNT_OPTS=""
 
-PASSTHROUGH_PATH=/home/nikratio/libfuse/build/example/passthrough_hp
-SCRATCH_SOURCE=/mnt/src/scratch
-TEST_SOURCE=/mnt/src/test
+export PASSTHROUGH_PATH=<path-to-libfuse-build>/example/passthrough_hp
```

Comment:
If this is now optional (which I think it is), can we add a comment that explains what the default value is? Is the following correrct?

```
# If this not set, attempt to execute mount.fuse.<helper_name> from $PATH
PASSTHROUGH_PATH=...
```

Reply:
Not exactly.
Mount helper to execute is determined from the -t argument to mount command,
So its going to be /sbin/mount.fuse.$FUSE_SUBTYP.
The fuse server binary to execute is attempted to be extracted from the mount helper name. 
So if the server binary can be found is $PATH and it's name is $FUSE_SUBTYPE,
No need to specify the server path explicitly.
In the local.config example, we can change the subtype to passthrough_hp

The downside of this approach is that a mount helper needs to be installed per subtype when testing different passthrough servers. 
Otoh maybe the right approach is to maintain different mount helpers per passthrough example because as you can see the source path argument is passed differently to passthrough_ll and I had to do something in the mount helper to work around that

Reply:
Can we add something like "If unset, defaults to looking for a binary named after the filesystem subtype in $PATH"? (I'd do these changes myself, but it seems I don't have edit permissions on the pull request).